Controller/DbControllers/PlaylistController.cs

```diff
@@ -12,7 +12,7 @@ namespace Controller.DbControllers
     public class PlaylistController: DbController<Playlist>
     {
 
-        public PlaylistController(DatabaseContext context) : base(context, context.Playlists)
+        public PlaylistController(DatabaseContext context, DbSet<Playlist> playlist) : base(context, playlist)
```

Comment:
Waarom doe je dit nou weer terug?

Reply:
REN NU HET NOG KAN

Reply:
REN!

Reply:
testen gingen fout ivm veranderde constructors

Reply:
oeps

Reply:
![image text](https://media.tenor.com/images/02eb4736b6fdf9aa3a9fb8c4bf8a16a0/tenor.gif)

Reply:
![image text](https://media1.tenor.com/images/1ac9362ead8907ae0c8f58f3a85e52b3/tenor.gif?itemid=7194861)

Reply:
Dan zorg je maar dat de testen weer werken :)
Die constructors horen geen Set mee te krijgen. Die horen ze zelf in die base() te gooien.

Reply:
klopt, daarom heb ik hem nu ff gerevert

Reply:
als het goed is werken ze weer